### PR TITLE
Load Carnival on the home page

### PR DESCRIPTION
**Pull request by pbrisbin**

### CODE REVIEW

File: Handler/Docs.hs [hunk 1]
```diff
@@ -1,19 +1,13 @@
 module Handler.Docs where
 
 import Import
+import Model.Site
 
 getDocsR :: Handler Html
 getDocsR = do
     root <- getAppRoot
-    Entity siteId _ <- runDB $ upsert (demoSite root) []
+    siteId <- runDB $ upsertDemoSite root
```
Comment (jferris) [inline Handler/Docs.hs]: We could make this into a `Widget` so the function/template pair can find the `siteId` on its own, right?

Comment (pbrisbin) [inline Handler/Docs.hs]: I'm not sure I follow, what is the template in the pair?

Comment (jferris) [inline Handler/Docs.hs]: The JavaScript to boot up the comments.

Comment (pbrisbin) [inline Handler/Docs.hs]: But the JavaScript is different between the two cases.

**Pull request merged**